Configuration/Test/test.py

```diff
@@ -0,0 +1,534 @@
+from SimG4Core.Application.hectorParameter_cfi import *
+import FWCore.ParameterSet.Config as cms
+import copy
+
+#
+# 09.08.17 patrykel
+# used to test new optics in 8_1_0
+#
+
+
+process = cms.Process("TestFlatGun")
+
+'''process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings',
+        'errors',
+        'infos',
+        'debugs'),
+    categories = cms.untracked.vstring('ForwardSim',
+        'TotemRP'),
+    debugModules = cms.untracked.vstring('*'),
+    errors = cms.untracked.PSet(
+        threshold = cms.untracked.string('ERROR')
+    ),
+    warnings = cms.untracked.PSet(
+        threshold = cms.untracked.string('WARNING')
+    ),
+    infos = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO')
+    ),
+    debugs = cms.untracked.PSet(
+        threshold = cms.untracked.string('DEBUG'),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        TotemRP = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        ),
+        ForwardSim = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        )
+    )
+)'''
+
+# Specify the maximum events to simulate
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1000)
+)
+
+# Configure the output module (save the result in a file)
+process.o1 = cms.OutputModule("PoolOutputModule",
+    outputCommands = cms.untracked.vstring('keep *'),
+    fileName = cms.untracked.string('file:test_1k.root')
+)
+process.outpath = cms.EndPath(process.o1)
+
+################## STEP 1 - process.generator
+process.source = cms.Source("EmptySource")
+
+# Use random number generator service
+#process.load("Configuration.TotemCommon.RandomNumbers_cfi")
+
+# particle generator paramteres
+process.load("Configuration.StandardSequences.Services_cff")
+
+process.load("Configuration.Generator.FlatLogKsiLogT_pp_13TeV_cfi")
+process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
+    g4SimHits = cms.PSet(initialSeed = cms.untracked.uint32(9876)),
+    SimG4Object = cms.PSet(initialSeed =cms.untracked.uint32(9876)),
+    RPSiDetDigitizer = cms.PSet(initialSeed =cms.untracked.uint32(137137)),
+    sourceSeed = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
+    generator = cms.PSet(initialSeed = cms.untracked.uint32(98766)),
+    SmearingGenerator = cms.PSet(initialSeed =cms.untracked.uint32(3849)),
+    T2Digis = cms.PSet(initialSeed =cms.untracked.uint32(98765)),
```

Comment:
Do we need seeds for T2Digis ? This module is not present

Reply:
Nowhere found any use of it so removed it.